playbooks/roles/minishift/tasks/main.yml

```diff
@@ -46,8 +49,7 @@
     dest: "{{ minishift_dest_dir }}/minishift.iso"
     timeout: 120
 
-# Check if the minishift profile exists
-- name: "Check if the minishift profile exists"
+- name: "Check if the minishift profile is running"
```

Comment:
We don't need to also check if the profile exists or by setting it active we accomplish that?